### DISCUSSION TITLE
Date & Time - Gregorian

### DISCUSSION THREAD

**Discussion by esoleyman** [Syriac-Localization/syriac-localization.github.io#2] [General]
For actual numbers, do we follow Arabic or Hebrew?

For example, at this time we used the letters to stand for the numbers similar to Arabic.

**quarter 1 wide** -- ܪܘܒܥܐ ܩܕܡܝܐ
**quarter 1 abbreviated** -- ܪܘܒܥܐ ܐ
**quarter 1 narrow** -- ܐ

or should we simplify and go with:

**quarter 1 wide** -- ܪܘܒܥܐ ܩܕܡܝܐ
**quarter 1 abbreviated** -- ܪܘܒܥܐ 1
**quarter 1 narrow** -- 1

**Comment by esoleyman** [comment]
For **Days - abbreviated**, **Days - narrow**, and **Days - short**, do we follow what Microsoft has which is essentially what we have now in CLDR or do we go a separate way?

Sun - ܚܕ
Mon - ܬܪܥܢ
Tue - ܬܠܬ
Wed - ܐܪܒܥ 
Thu - ܚܡܫ
Fri - ܥܪܘ
Sat - ܫܒܬܐ

**Reply by SargisYonan** [reply]
I like this idea

**Reply by SargisYonan** [reply]
<img width="412" alt="Screen Shot 2023-06-22 at 10 46 25 PM" src="https://github.com/Syriac-Localization/syriac-localization.github.io/assets/5349027/311107d1-16cc-49de-9ee9-7960f3e17e2f">
we actually seem to have this already

**Reply by esoleyman** [reply]
Then let's keep it as is

**Comment by SargisYonan** [comment]
> For actual numbers, do we follow Arabic or Hebrew?
> 
> For example, at this time we used the letters to stand for the numbers similar to Arabic.
> 
> **quarter 1 wide** -- ܪܘܒܥܐ ܩܕܡܝܐ **quarter 1 abbreviated** -- ܪܘܒܥܐ ܐ **quarter 1 narrow** -- ܐ
> 
> or should we simplify and go with:
> 
> **quarter 1 wide** -- ܪܘܒܥܐ ܩܕܡܝܐ **quarter 1 abbreviated** -- ܪܘܒܥܐ 1 **quarter 1 narrow** -- 1

I agree. We should map `0-9` to `0-9`. Let's leave cardinality to things like "first", "second", etc., not 1, 2, 3, etc.

**Reply by SargisYonan** [reply]
what about "1st Quarter". I think ܪܘܒܥܐ ܩܕܡܝܐ is appropriate here

**Reply by esoleyman** [reply]
I agree for the wide formatting but what about narrow and abbreviated formatting?

**Current formatting**

_Narrow_:
Q1: ܪܘܒܥܐ ܐ
Q2: ܪܘܒܥܐ ܒ
Q3: ܪܘܒܥܐ ܓ 
Q4: ܪܘܒܥܐ ܕ

_Abbreviated_:
1: ܐ
2: ܒ
3: ܓ
4: ܕ

**Proposed formatting**

Narrow:
Q1: ܪܘܒܥܐ 1
Q2: ܪܘܒܥܐ 2
Q3: ܪܘܒܥܐ 3
Q4: ܪܘܒܥܐ 4

Abbreviated:
1: 1
2: 2
3: 3
4: 4

**Reply by SargisYonan** [reply]
I agree, but do you think the abbreviated would make sense?

**Reply by esoleyman** [reply]
Arabic uses 1, 2, 3, and 4 for the Abbreviated as well
